examples/example_curve.py

```diff
@@ -15,13 +15,13 @@
 y = np.sin(theta)
 z = np.linspace(0, 1, n)
 print("Helix Data")
-curve = Curve(x=x, y=y, z=z, k=4, Nctl=16, niter=100)
+curve = Curve(x=x, y=y, z=z, k=4, tl=16, ter=100)
```

Comment:
Thanks for the quick fix. I think there was a find and replace gone wrong here.

Reply:
fixed